instat/dlgTransformText.vb

```diff
@@ -357,26 +360,22 @@ Public Class dlgTransformText
             ucrNudFirstWord.Visible = True
             ucrNudLastWord.Visible = True
             If ucrChkFirstOr.Checked Then
-                ucrBase.clsRsyntax.RemoveParameter("start")
                 ucrReceiverFirstWord.SetMeAsReceiver()
-                ucrBase.clsRsyntax.AddParameter("start", clsRFunctionParameter:=ucrReceiverFirstWord.GetVariables())
+                clsSubstringFunction.AddParameter("start", clsRFunctionParameter:=ucrReceiverFirstWord.GetVariables(), iPosition:=1)

```

Comment:
Adding this parameter to the `cls` not `ucrbase.clsrsyntax`. Therefore we won't have to run `.RemoveParameter`